### DISCUSSION TITLE
mtrack file bulk export or process?

### DISCUSSION THREAD

**Discussion by cooperrc** [andreynovikov/trekarta#291] [Q&A]
I've been using Trekarta for almost 2 years now. Its an amazing app for maps and tracking biking/hiking, thanks!

Is there a way to process mtrack files outside of trekarta? 

The use-case I'm considering is getting lat-long and speed from recorded tracks in bulk. 

Right now, I think the only way would be to individually _share_, _to GPX_, then process from there. 

Is there a Python/other language that could read and process `*.mtrack` files?

**Comment by Willem3141** [comment]
I had the same problem where I wanted to bulk-convert .mtrack files to GPX. [Here](https://gist.github.com/Willem3141/0d92440a1c7ad7fed956c49cd950b6d4) is the simple Python script I wrote to do this.

**Reply by cooperrc** [reply]
Cool! I'll give it a shot.
